srv/runner/runner.py

```diff
@@ -66,11 +73,18 @@ def run_job(self, job):
         Runs job
 
         """
+        # TODO: temporary solution, replace with the additional job in the pipeline?
+        # Download file and replace file URI param with local path
+        s3_uri = job.input_parameters[0]
```

Comment:
Here, I think we need to check if have anything in input_parameters. Otherwise it might be exception if there are no input parameters.